tools/integration_tests/operations/operations_test.go

```diff
@@ -181,13 +184,19 @@ func TestMain(m *testing.M) {
 		// By default, creating emptyFile is disabled.
 		{"--experimental-enable-json-read=true", "--implicit-dirs=true"}}
 
-	if !testing.Short() {
+	if !testing.Short() && !setup.TestOnTPCEndPoint() {
 		flagsSet = append(flagsSet, []string{"--client-protocol=grpc", "--implicit-dirs=true"})
 	}
 
 	mountConfigFlags := createMountConfigsAndEquivalentFlags()
 	flagsSet = append(flagsSet, mountConfigFlags...)
 
+	// gRPC tests will not run in TPC environment
```

Comment:
Move this comment to the place where you are actually guarding grpc flag.

Please correct the comment - // Only running static_mounting test for TPC.